em/em.go

```diff
@@ -0,0 +1,145 @@
+package em
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"log"
+	"log/slog"
+	"strings"
+
+	"github.com/AudiusProject/audius-d/acdc"
+	"github.com/AudiusProject/audius-d/hashes"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/spf13/cobra"
+)
+
+var EmCmd *cobra.Command
+
+const (
+	keyfileLocation = ".audius_developer_app_private_key"
```

Comment:
Is this something that could be eventually stored in the ContextConfig? (I would be interested in understanding more about the use case so that we can provide better abstraction in the future)